deploy/awstools/awstools.py

```diff
@@ -113,18 +122,10 @@ def get_f1_ami_id():
     return response['Images'][0]['ImageId']
 
 def get_aws_userid():
-    """ Get the user's IAM ID to intelligently create a bucket name when doing managerinit.
-    The previous method to do this was:
-
+    """ Get the user's IAM ID to intelligently create a bucket name when doing managerinit. """
```

Comment:
This doesn't work if you are not in EC2... maybe just return a dummy string?

Reply:
Or fall back to using `os.getenv('USER')`?